src/Visualization/Python/PlotTrajectories.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Plot XY trajectory from inspiral run, concatenating all segments
+
+import glob
+import os
+
+import click
+import h5py
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+@click.command()
```

Comment:
add `name="trajectories"` so it appears properly in help texts.

Also name the function `plot_trajectories_command` for consistency.